unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/CylinderShape/Handler/ECSCylinderShapeComponentHandler.cs

```diff
@@ -42,11 +47,21 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBCyl
 
         private void GenerateRenderer(Mesh mesh, IParcelScene scene, IDCLEntity entity, PBCylinderShape model)
         {
+            GenerateHolder(entity);
             meshesInfo = ECSComponentsUtils.GenerateMeshInfo(entity, mesh, entity.gameObject, model.Visible, model.WithCollisions, model.IsPointerBlocker);
 
             // Note: We should add the rendereable to the data store and dispose when it not longer exists
             rendereable = ECSComponentsUtils.AddRendereableToDataStore(scene.sceneData.id, entity.entityId, mesh, entity.gameObject, meshesInfo.renderers);
         }
+        
+        private void GenerateHolder(IDCLEntity entity)
+        {
+            if(meshHolderGameObject != null)
+                GameObject.Destroy(meshHolderGameObject);
+            
+            meshHolderGameObject = new GameObject();
```

Comment:
Is it possible to avoid the destruction and recreation of `meshHolderGameObject` in every component update?

Reply:
Completly removed the game object